evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -86,8 +89,7 @@ def validate(self, X, y):
                                                   data_check_name=self.name,
                                                   message_code=DataCheckMessageCode.CLASS_IMBALANCE_BELOW_FOLDS,
                                                   details={"target_values": below_threshold_values}), results)
-
-        counts = fold_counts / fold_counts.sum()
+        counts = fold_counts / (fold_counts + fold_counts.values[0])
```

Comment:
Oh wow, this was the only code change needed?! Very cool 😎 

This logic assumes that `fold_counts` is sorted from the highest to lowest count. For clarity/redundancy, I suggest you set `sort=True` in the call to `value_counts` on line 80. [`True` is the default for `sort`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.value_counts.html), but for code comprehension I think its helpful to have.

What if the target has `nan`s -- do we have test coverage for this case? I think the impl is all set because `dropna` defaults to `False`, but it would be good coverage to add.